python/lib/sift_py/grpc/transport.py

```diff
@@ -126,25 +138,35 @@ def _compute_channel_options(opts: Optional[SiftChannelConfig] = None) -> List[T
     return options
 
 
-def _metadata_interceptor(config: SiftChannelConfig) -> ClientInterceptor:
+def _metadata_interceptor(
+    config: SiftChannelConfig, metadata_kwargs: Optional[Dict[str, Any]] = None
+) -> ClientInterceptor:
     """
     Any new metadata goes here.
     """
     apikey = config["apikey"]
-    metadata: Metadata = [
-        ("authorization", f"Bearer {apikey}"),
-    ]
+
+    kwargs = []
+    if metadata_kwargs:
+        kwargs = [(key, val) for key, val in metadata_kwargs.items()]
+
+    metadata: Metadata = [("authorization", f"Bearer {apikey}"), *kwargs]
```

Comment:
Could also declare the `metadata` list first then append if `metadata_kwargs`. Avoids the additional list allocation then having to spread.

Reply:
👩‍🍳 🤌